cognite/neat/rules/importers/_dms2rules.py

```diff
@@ -92,6 +92,8 @@ def to_rules(
                             direct_value_type = "UNKNOWN"
                         else:
                             direct_value_type = ViewPropEntity.from_id(prop.source)
+
+                        print(direct_value_type, type(direct_value_type))
```

Comment:
Debug?
```suggestion
```

Reply:
Hm I do not have that locally interesting